lib/ovirt/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ovirt
-  VERSION = "0.7.2"
+  VERSION = "0.7.3"
```

Comment:
Please don't make any changes to the version file.  I will update it on release.